src/core/components/array-model.jsx

```diff
@@ -7,6 +7,7 @@ export default class ArrayModel extends Component {
   static propTypes = {
     schema: PropTypes.object.isRequired,
     getComponent: PropTypes.func.isRequired,
+    getConfigs: PropTypes.func.isRequired,
```

Comment:
I'm not a huge fan of the `getConfigs` handaround - would prefer to see a container component mapping from the configs to props for children.